src/libsyntax/print/pprust.rs

```diff
@@ -1083,7 +1083,7 @@ impl<'a> State<'a> {
             }
             ast::ForeignItemKind::Static(ref t, m) => {
                 self.head(visibility_qualified(&item.vis, "static"));
-                if m == ast::Mutability::Mutable {
+                if m == ast::Mutability::Mut {
```

Comment:
all these comparisons make me wonder if we should have `is_mut()` and `is_immut()` helper methods

Reply:
I would suggest just nixing `PartialEq` by using the beautiful `if let` instead.